internal/jobobject/jobobject.go

```diff
@@ -68,6 +72,9 @@ type Options struct {
 	// `UseNTVariant` specifies if we should use the `Nt` variant of Open/CreateJobObject.
 	// Defaults to false.
 	UseNTVariant bool
+	// `Silo` specifies to promote the job to a silo. This additionally sets the flag JOB_OBJECT_LIMIT_KILL_ON_JOB_CLOSE as it is required for
```

Comment:
There is a sweet little vs code plugin Rewrap that makes comments like this not look so awful and maintains the line ending at 80 chars by default I believe.

Reply:
You're telling me I can stop manually doing this???? 😆 I had this change queued up, but was fixing some CI breaks before undrafting this again. Grabbin the plugin though, looks great

Reply:
Resolving as this is wrapped now